man/systemd.netdev.xml

```diff
@@ -1007,6 +1007,18 @@
         <xi:include href="version-info.xml" xpointer="v247"/>
         </listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>MultiProto=</varname></term>
+        <listitem>
+          <para>Takes a boolean value. When true, activates support for protocols similar to the one specified by ethertype.
```

Comment:
"ethertype", what does that refer to? please explain this briefly in the sentence